### PR TITLE
Added Hobby classifier and rough path forward for future classifiers

### PR DESCRIPTION
**Pull request by johntosberg**
There's probably more kotlin-y and better OO ways to do this, but it's Monday morning so I took some lazy approaches for now. Path to make it smarter is probably pretty obvious at this point, I'll probably pick it back up later

### CODE REVIEW

File: src/main/kotlin/blackmichael/discord/command/classifier/SubjectClassifier.kt [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    override val definition = Definition(
+            name = "Is this a potato?",
+            description = "Determines if something is, or is not.",
+            usage = "is <subject> a (potato|hobby)?"
+    )
+
+    override fun EnhancedEventListener.run() {
+        commandSet.command("is") {
+            val isPotatoClassifier = content.removePrefix("${commandSet.prefix}is ").contains("potato")
+            val isHobbyClassifier = content.removePrefix("${commandSet.prefix}is ").contains("potato")
```
Comment (blackmichael) [inline src/main/kotlin/blackmichael/discord/command/classifier/SubjectClassifier.kt]: Copy pasta error, I think you want `contains(“hobby”)`

File: src/main/kotlin/blackmichael/discord/command/classifier/SubjectClassifier.kt [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        private val botId: String
+) : Command(eventListener, commandSet) {
+
+    val snarkyResponses = listOf<String>(
+            "You cannot do that yet.",
+            "I'm trying my best, but my creators failed me",
+            "Maybe someday I'll know the answer. Today is not that day")
+
+    override val definition = Definition(
+            name = "Is this a potato?",
```
Comment (blackmichael) [inline src/main/kotlin/blackmichael/discord/command/classifier/SubjectClassifier.kt]: Maybe “is this a thing?”

**Review by blackmichael — COMMENTED** [review]
I don’t have CI yet but you need to run ./gradlew  ktlintFormat or else Heroku fails

File: src/main/kotlin/blackmichael/discord/command/classifier/Classifier.kt [hunk 3]
```diff
@@ -0,0 +1,15 @@
+package blackmichael.discord.command.classifier
```
Comment (DiabetesInc) [inline src/main/kotlin/blackmichael/discord/command/classifier/Classifier.kt:1]: Big fan of this package structure

**Pull request merged**